src/tile_source.js

```diff
@@ -51,42 +38,42 @@ export default class TileSource {
     }
 
     // Mercator projection
-    static projectTile (tile) {
+    static projectData (source) {
         var timer = +new Date();
-        for (var t in tile.layers) {
-            var num_features = tile.layers[t].features.length;
+        for (var t in source.layers) {
+            var num_features = source.layers[t].features.length;
             for (var f=0; f < num_features; f++) {
-                var feature = tile.layers[t].features[f];
+                var feature = source.layers[t].features[f];
                 feature.geometry.coordinates = Geo.transformGeometry(feature.geometry, Geo.latLngToMeters);
             }
         }
 
-        if (tile.debug !== undefined) {
-            tile.debug.projection = +new Date() - timer;
+        if (source.debug !== undefined) {
+            source.debug.projection = +new Date() - timer;
```

Comment:
Since we have multiple places like this where we have a `source` object to put debug info on but we don't have a `tile` object, we should probably just sum (for those where sum is appropriate) all source debug values and roll them up into the tile.